server/tserver/src/main/java/org/apache/accumulo/tserver/tablet/Tablet.java

```diff
@@ -1597,4 +1597,12 @@ public void setLastAccessTime() {
     this.lastAccessTime = System.nanoTime();
   }
 
+  public boolean isInUse() {
```

Comment:
The activeScan set is a hashset and writesInProgress is not volatile.  To go with the rest of the tablet code, need to sync to ensure getting latest data.

```suggestion
  public synchronized boolean isInUse() {
```